proposals/xxx-media-service-data-progress-bar-improvements.md

```diff
@@ -0,0 +1,48 @@
+# Add Currently Playing Media Image to MediaServiceData
+
+* Proposal: [SDL-xxx](xxx-media-service-data-progress-bar-improvements.md)
+* Author: [Jack Byrne](https://github.com/jacklivio)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / RPC]
+
+## Introduction
+
+This proposal adds new paramters to the media service data to allow the app consumers know the playback status and progress of a selected track.
+
+## Motivation
+
+The movitation for the proposal came from creating an implementation of a media type app service consumer that displays the progress bar data from an active media service provider.
+
+Currently app service providers can send notify consumers about a tracks playback progress via `trackPlaybackProgress` and `trackPlaybackDuration`. However, if an app consumer wants to display an updating progress using these parameters, it would require the app provider to send an OnAppServiceData RPC with updated information every second.
```

Comment:
```suggestion
Currently app service providers can notify consumers about a track's playback progress via `trackPlaybackProgress` and `trackPlaybackDuration`. However, if an app consumer wants to display progress using these parameters, it would require the app provider to send an `OnAppServiceData` RPC with updated information every second.
```